### DISCUSSION TITLE
Two typos

### DISCUSSION THREAD

**Discussion by bewuethr** [actions/importer-labs#44] [Feedback]
Hi, I was working through the Jenkins lab, and noticed two small issues:

- There is a job called `test_mutlibranch_pipeline` (typo "mutli")
- In [lab 6](https://github.com/actions/importer-labs/blob/main/jenkins/6-migrate.md), the source URL [here](https://github.com/actions/importer-labs/blob/main/jenkins/6-migrate.md#performing-a-migration) is shown as `http://localhost:8080/monas_dev_work/job/monas_freestyle`, but it should be `http://localhost:8080/job/monas_dev_work/job/monas_freestyle` (missing `job/`). The commands below have it right, but I first tried editing a previous command with the incorrect URL and was confused

**Comment by Sensationalshelly8** [comment]
Thank you I will make sure to make the proper corrections to the issues found. I appreciate the feedback.
